SpotifyAPI.Web/Models/Request/LibrarySaveEpisodesRequest.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections.Generic;
+
+namespace SpotifyAPI.Web
+{
+  public class LibrarySaveEpisodesRequest : RequestParams
+  {
+    /// <summary>
+    ///
+    /// </summary>
+    /// <param name="uris">A comma-separated list of Spotify URIs for the shows to be added to the user’s library.</param>
+    public LibrarySaveEpisodesRequest(IList<string> uris)
+    {
+      Ensure.ArgumentNotNullOrEmptyList(uris, nameof(uris));
+
+      URIs = uris;
+    }
+
+    /// <summary>
+    /// A comma-separated list of Spotify URIs for the shows to be added to the user’s library.
+    /// </summary>
+    /// <value></value>
+    [QueryParam("uris")]
+    public IList<string> URIs { get; }
```

Comment:
I think this should be `ids` instead of `uris`. See: https://developer.spotify.com/documentation/web-api/reference/#endpoint-save-episodes-user